lib/Parser/Hash.h

```diff
@@ -76,8 +76,6 @@ struct PidRefStack
     bool IsModuleExport() const { return isModuleExport; }
     void SetModuleExport()    { isModuleExport = true; }
 
-    void TrackAssignment(charcount_t ichMin, charcount_t ichLim);
```

Comment:
`PidRefStack::isAsg` and `PidRefStack::span` look like they are also dead now, never set to anything but the constructor initialized values.